asf_core_data/pipeline/mcs/test/test_process_mcs_installations.py

```diff
@@ -104,16 +87,21 @@ def test_regex_extraction_works():
 
 def test_same_lines_before_after():
     """
-    Checks if raw and processed datasets have the same number of lines.
+    Checks if raw and processed datasets have the same number of lines
+    (discounting the lines corresponding to test accounts and duplicate lines).
     """
-    assert len(raw_installations) == len(processed_installations)
+
+    assert len(
+        drop_instances_test_accounts(
+            raw_installations, "Installation Company Name"
+        ).drop_duplicates()
+    ) == len(processed_installations)
 
 
 def test_no_duplicate_lines():
     """
-    Checks that there are no duplicate lines in either raw or processed datasets
+    Checks that there are no duplicate lines in the processed dataset.
     """
-    assert len(raw_installations.drop_duplicates()) == len(raw_installations)
     assert len(processed_installations.drop_duplicates()) == len(
```

Comment:
hey @athrado,
- there's another test for the columns, it actually tests more than just the number of columns (tests the names);
- Regarding the use of `len()` vs. `shape[0]`, they do the exact same thing which is returning the number of lines. I don't find anything about using `shape[0]` being best practice instead of using `len()` - could you point me to any resource on this? I actually find `len()` more readable as people might mistake `shape[0]` by `shape[1]`. Here's a [stackoverflow post](https://stackoverflow.com/questions/15943769/how-do-i-get-the-row-count-of-a-pandas-dataframe) with multiple answers comparing `len()` and `shape`.